QMLComponents/boundcontrols/boundcontrolbase.cpp

```diff
@@ -33,7 +33,7 @@ Json::Value BoundControlBase::createMeta()  const
 { 
 	Json::Value meta(Json::objectValue);
 	
-	if (_isColumn || _control->encodeValue())	
```

Comment:
What is the case when encodeValue is true but this is not a column?

Reply:
Im removing _isColumn from there anyway

Reply:
The case is: the control says it wants to be encoded and so it is encoded.
Just look up whatever sets _encodeValue...